resources/components/CardStack.vue

```diff
@@ -47,19 +50,23 @@ module.exports = {
 			return this.images[ this.queue ];
 		},
 
-		/**
-		 * @return {Object|undefined}
-		 */
-		currentImage: function () {
-			return this.imagesInQueue[ 0 ];
-		},
-
 		/**
 		 * Pending state is queue-specific
 		 * @return {bool}
 		 */
 		isPending: function () {
 			return this.pending[ this.queue ];
+		},
+
+		/**
+		 * Whether to render the ImageCard.
+		 * We need a dedicated computed property for this because
+		 * ResourceLoader can't handle "&&" in the template.
```

Comment:
Hah, I noticed this yesterday and wondered if it was a Vue thing or a RL thing. I've gotten some unfortunately vague errors in the console for stuff like this which is difficult to debug...